views/components/globals.html

```diff
@@ -12,37 +12,23 @@
 
 {% block head -%}
     <!-- Google Tag Manager -->
-    {% if environment === 'prod' %}
-        <!-- Live analytics -->
-        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-            new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-            j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-            'https://www.googletagmanager.com/gtm.js?id='+i+dl+ '&gtm_auth=UuZZs4BGR3tNzCze2MIbBw&gtm_preview=env-2&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);
-          })(window,document,'script','dataLayer','GTM-T56C5T7');</script>
-    {% else %}
-        <!-- Dev analytics -->
-        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-            new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-            j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-            'https://www.googletagmanager.com/gtm.js?id='+i+dl+ '&gtm_auth=uYQEEAOU5-Kwl3yix5HG_A&gtm_preview=env-87&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);
-          })(window,document,'script','dataLayer','GTM-T56C5T7');</script>
-    {% endif %}
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Same here, please format the code

Reply:
it's copy and paste direct from what google provides. Formatting will make it harder to check if its changed in future against what google provides?